packages/core/crates/core-path/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "core-path"
+version = "0.1.0"
+authors = ["HOPR Association <tech@hoprnet.org>"]
+edition = "2021"
+description = "Contains mixnet path construction and path finding logic"
+homepage = "https://hoprnet.org/"
+repository = "https://github.com/hoprnet/hoprnet"
+license = "GPL-3.0-only"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["console_error_panic_hook", "wasm"]
+wasm = [ "dep:wasm-bindgen" ]
+console_error_panic_hook = [ "dep:console_error_panic_hook" ]
+
+[dependencies]
+hex = "0.4.3"
+js-sys = "0.3.63"
+libp2p-identity = { version = "0.2.0", features = ["peerid"] }
```

Comment:
We could probably move such dependencies which we use all over the place into the top-level workspace configuration and refer to them here to reduce version mismatches later.